GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionGroupRestService.java

```diff
@@ -33,7 +33,7 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.waterforpeople.mapping.app.gwt.client.survey.QuestionGroupDto;
-import org.waterforpeople.mapping.app.util.DtoMarshaller;
+import org.waterforpeople.mapping.app.gwt.client.survey.QuestionGroupDtoMapper;
```

Comment:
I just noticed the package of this as well as the `QuestionDtoMapper` and I think we should put it somewhere else.... e.g. `org.akvo.flow.domain.mapper.`  please propose some other one but I think it should not be in the `...gwt.client..`  because these are packages we have been slowly phasing out as they were part of the google web toolkit